pkg/iam/template.go

```diff
@@ -45,14 +47,21 @@ func isChinaRegion(region string) bool {
 	return strings.Contains(region, "cn-")
 }
 
-func getInlinePolicyTemplate(roleType string) string {
+func getInlinePolicyTemplate(roleType string, objectLabels map[string]string) string {
 	switch roleType {
 	case BastionRole:
 		return bastionPolicyTemplate
 	case ControlPlaneRole:
 		return controlPlanePolicyTemplate
 	case NodesRole:
-		return nodesTemplate
+		if _, ok := objectLabels[AWSReducedInstanceProfileIAMPermissionsForWorkersLabel]; ok {
```

Comment:
I find it a bit confusing that an `AWSCluster` could have `alpha.aws.giantswarm.io/reduced-instance-permissions-workers: "false"`, and that would mean using reduced permissions, as we are only checking existence of the label. What do you think about checking the value too, and only using reduced permissions if the value is true?

Reply:
Makes sense, done